content/context-resource/index.md

```diff
@@ -0,0 +1,320 @@
+  , rootHash : Hash
+  , schema : Schema
+  , stats : Stats
+  , status : Status
+  , title : Maybe String
+  }
+```
+
+See each section below of the explanation for each attribute.
+
+
+#### Id
+
+* Type: Name (This is defined in the specification as `[a-z][a-z0-9-]*`, the
+  future specification will make it more clear).
+
+The register identifier. The current implications for this value make it act
+as a unique identifier for the environment is belongs to. This RFC doesn't
+pretend to change this fact.
+
+
+#### Copyright
+
+* Type: String
+
+The register copyright. E.g. `© Crown copyright`. Similar to the `copyright`
+field in the Register register but required to be informed.
+
+
+#### Custodian
+
+* Type: String
+
+The same value currently exposed in the Register resource.
+
+
+#### Description
+
+* Type: Optional String
+
+The human readable description of the register.
+
+
+#### Hashing algorithm
+
+* Type: HashingAlgorithm
+
+The hashing algorithm used throughout the register.
+
+```elm
+type HashingAlgorithm =
+  { id : Name
+  , functionType : Byte
+  , digestLength : Byte
+  }
+```
+
+This depends on RFC0013
+
+|Name|Description|
+|-|-|
+|`digest-length`|The length of the digest in bytes. E.g. `0x12`|
+|`function-type`|The identifier of the hash function. E.g. `0x20`|
+|`id`| The name of the algorithm. E.g. `sha2-256`|
+
+See also: [Multihash table function type
+identifiers](https://github.com/multiformats/multihash/blob/master/hashtable.csv).
+
+
+#### Licence
+
+* Type: String (This could get restricted if/when we find a suitable list to
+  restrict the field)
+
+#### Root hash
+
+* Type: Hash
+
+The root hash for the register. Note that signatures will be addressed in
+another RFC. This is not strictly part of the metadata, it is derived from the
+data.
+
+#### Schema
+
+* Type: Schema
+
+The set of attributes that define the data allowed in the register.
+
+```elm
+type Attribute =
+  { id : Name
+  , datatype : Datatype
+  , title : Maybe String
+  , description : Maybe Text
+  }
+
+type Schema =
+  Set Attribute
+```
+
+|Name|Description|
+|-|-|
+|Id|Attribute identifier (e.g. `start-date`)|
+|Datatype|Datatype identifier (e.g. `datetime`)|
+|Title|Human readable attribute name (e.g. `Start date`)|
+|Description|Human readable attribute description (e.g. `The date a record first became relevant to a register.`)|
+
+
+#### Stats
+
+* Type: Stats
+
+The summary of objects stored in the register. This overlaps with the Register
+resource and it is not strictly part of the metadata, it is derived from the data.
+
+
+#### Status
+
+* Type: Status
+
+The status of the register. Either active or retired. This addresses the
+problem of being able to retire an entire register (see
+[issue #17](https://github.com/openregister/registers-rfcs/issues/17)).
+
+```elm
+type Status
+  = Active { startDate : Timestamp }
+  | Retired { startDate : Timestamp
+            , endDate : Timestamp
+            , replacement : Maybe Url
```

Comment:
I think this is the first time registers have linked to other registers by URL.

This could cause problems if that URL breaks but the register is still available elsewhere.

Another way of doing this would be to use a CURIE then rely on a catalog to resolve this to something (and this could be a catalog outside of GDS's control if GDS is no longer hosting the register register)

Reply:
Fair enough 👍

Reply:
Right now I cannot see where this information would be persisted? Is there a companion to this that addresses how this is represented in RSF/Entry log? If not I think this should be pulled out into a separate RFC that describes Status and its representation.

Reply:
@arnau please can you address this [comment](https://github.com/openregister/registers-rfcs/pull/31/files#r225571551)?

Reply:
@gidsg status must be part of the register metadata which means it has to be encoded in RSF. The way to express that will highly depend on the metadata solution we end up having but, if it is in line with what we have right now it will be an system entry with the appropriate key pointing to a blob with these information.